packages/docs-reanimated/src/examples/RelativeCoords.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import Animated, {
+  useAnimatedRef,
+  getRelativeCoords,
+} from 'react-native-reanimated';
+import { Gesture, GestureDetector } from 'react-native-gesture-handler';
+
+const RelativeCoords = () => {
+  const animatedRef = useAnimatedRef();
+  const [coords, setCoords] = useState({ x: 0, y: 0 });
+  const [boxCoords, setBoxCoords] = useState({ x: 0, y: 0 });
+
+  const tapGesture = Gesture.Tap().onEnd((event) => {
+    const relativeCoords = getRelativeCoords(
+      animatedRef,
+      event.absoluteX,
+      event.absoluteY
+    );
+    setBoxCoords({ x: event.x, y: event.y });
+    if (relativeCoords) {
+      setCoords(relativeCoords);
+    }
+  });
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.coordsData}>Relative coordinates to parent:</Text>
+      <Text style={[styles.coordsData, styles.coords]}>
+        x={coords.x.toFixed()} y=
+        {coords.y.toFixed()}
+      </Text>
+      <Text style={styles.coordsData}>Coordinates inside box:</Text>
```

Comment:
`Relative coordinates to parent:` and `Coordinates inside box:` don't describe them well and could be confusing. Can you try to find better names? Unfortunately, I don't have any good suggestions here :/